docs/fidesops/docs/tutorial/execute_privacy_request.md

```diff
@@ -59,7 +59,7 @@ if __name__ == "__main__":
     email = "user@example.com"
     privacy_requests = create_privacy_request(
         email=email,
-        policy_key="example-request-policy",
+        policy_key="example_request_policy",
```

Comment:
Related privacy request guide https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/guides/privacy_requests.md#how-can-i-execute-a-privacy-request

has a sample request referencing a policy key that needs to be snake-cased